nfv-example-cnf-catalog.Dockerfile

```diff
@@ -9,3 +9,4 @@ ADD nfv-example-cnf-catalog /configs
 # Set DC-specific label for the location of the DC root directory
 # in the image
 LABEL operators.operatorframework.io.index.configs.v1=/configs
+LABEL maintainer="Telco DCI team" quay.expires-after=5h
```

Comment:
`quay.expires-after=5h` is that how long the catalog will be available?

Reply:
yeah for the image:tag, well... been this a public artifact that is pushed to the public quay may be good to do not to remove it.
Good catch, let me remove the label. Thanks